generator/js_of_ast.ml

```diff
@@ -1142,8 +1127,8 @@ and js_of_expression (sm : shadow_map) ctx dest e =
       else begin (* rename the constructor to remove "" prefix *)
         let id2 = 
           match p.txt with
-          | Longident.Lident s -> Longident.Lident (rename_constructor s)
-          | Longident.Ldot(l, s) -> Longident.Ldot(l, rename_constructor s)  
+          | Longident.Lident s -> Longident.Lident s
+          | Longident.Ldot(l, s) -> Longident.Ldot(l, s 
```

Comment:
Syntax error here.

Reply:
Sorry, i thought i had removed all the calls. Thanks for the review.